src/test/ui-fulldeps/resolve-error.rs

```diff
@@ -35,46 +35,29 @@ macro_rules! attr_proc_mac {
 }
 
 #[derive(FooWithLongNan)]
-//~^ ERROR cannot find derive macro `FooWithLongNan` in this scope
```

Comment:
I think it's better to keep these annotations.
UI and compile-fail tests are going to be unified and `ERROR`/`WARN` will be necessary again.